src/partials/hero.html

```diff
@@ -1,2 +1,23 @@
-<!-- Loads the specified .html file // todo: include header.html -->
-<load="partials/header.html" />
\ No newline at end of file
+<!-- Hero Section -->
```

Comment:
```suggestion
<section id="hero" class="section">
  <div class="container hero-content">
    <div class="hero-section">
      <h1 class="hero-text">
        Discover the joy of learning
        <span class="hero-english">English</span> with us!
      </h1>
      <button onclick="location.href='#form'" class="button hero-button">
        Leave an application
      </button>
      <div class="hero-checkmark">
        <div class="hero-checkmark-container">
          <svg class="hero-svg">
            <use
              class="hero-use"
              xlink:href="../img/icons.svg#icon-checkmark"
            ></use>
          </svg>
        </div>
      </div>
    </div>
  </div>
</section>

```

Reply:
The idea of hero-section is to have background image. It is not possible to put hero-section inside of container since container have paddings and currently margins. With such approach background image will not be filling the area according to figma.